### PR TITLE
Lingeswar adduser

### PR DESCRIPTION
**Pull request by Lingeswar03**
add user

### CODE REVIEW

File: src/main/java/com/vagdeviitsol/fitwala/FitwalaService.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+@Service
+public class FitwalaService {
+	
+	@Autowired
+	private FRepository FitwalaRepo;
+	
+	
+	public FUsers addDetails(FUsers user) {
```
Comment (vagdeviitsolutions18) [inline src/main/java/com/vagdeviitsol/fitwala/FitwalaService.java:14]: change method name to addingDetails

**Review by vagdeviitsolutions18 — COMMENTED** [review]
change details

File: src/main/java/com/vagdeviitsol/fitwala/FitwalaService.java (hunk 1 above)
Comment (Lingeswar03) [reply src/main/java/com/vagdeviitsol/fitwala/FitwalaService.java:14]: changed as requested

**Comment by Lingeswar03** [comment]
review comments resolved

**Comment by vagdeviitsolutions18** [comment]
add user functionality merged

**Pull request merged**